shows/736 - CJ Reynolds Is Joining Syntax.md

```diff
@@ -0,0 +1,72 @@
+
+### Show Notes
+
+* **[00:00](#t=00:00)** Welcome to Syntax!
+* **[01:06](#t=01:06)** Brought to you by [Sentry.io](www.sentry.io).
+* **[04:39](#t=04:39)** What got you into web development?
+* **[09:23](#t=09:23)** Does a degree in computer science help you in your day to day as a web developer?
+* [Bell Curve Meme](https://x.com/_hariti/status/1761059582229573895?s=20)
+* **[11:39](#t=11:39)** Wes downloads every single .com domain name.
+* **[15:19](#t=15:19)** What was your very first web development job?
+* **[18:12](#t=18:12)** How did you go from C# to websites?
+* **[21:12](#t=21:12)** What is the worst thing you've had to do as a developer?
+* **[22:31](#t=22:31)** What are some of your favorite projects?
+* [Is It New Years Day](https://isitnewyearsday.com/)
+* [Dung Hero](https://dunghero.online/)
+* **[26:32](#t=26:32)** What tech are you into right now? What inspires you?
+* **[27:52](#t=27:52)** Why do you like experimenting with new technologies?
+* **[29:39](#t=29:39)** [Hono v4.0.5](https://hono.dev/).
+* **[30:29](#t=30:29)** [HTMZ](https://leanrada.com/htmz/)
+* **[31:44](#t=31:44)** If you were to build a startup today, what would your stack be?
+* **[32:39](#t=32:39)** What's your tech setup?
+* [Coding Garden GitHub](https://github.com/codingGarden/vscode-settings)
+* [RK Royal Kludge Keyboard](https://www.amazon.com/RK-ROYAL-KLUDGE-Bluetooth-Mechanical/dp/B08JCQ9JMQ/)
+* **[37:51](#t=37:51)** What got you into live-streaming?
+* **[40:13](#t=40:13)** Why do you use a blue screen?
+* **[41:19](#t=41:19)** What's your philosophy on teaching?
+* [Bloom's Taxonomy](https://en.wikipedia.org/wiki/Bloom's_taxonomy)
+* **[43:56](#t=43:56)** What are your non-developer hobbies?
+* [CJ Skateboarding with Syntax deck](https://twitter.com/coding_garden/status/1749488479887561078)
+* [CJ's floofy doggo, Pig](https://www.instagram.com/pigthesaint/)
+* [Sony VX10000](https://www.youtube.com/watch?v=qAhR1aWwLG0)
+* [Sony Mavica](https://en.wikipedia.org/wiki/Sony_Mavica)
+* **[51:00](#t=51:00)** Sick Picks + Shameless Plugs.
+
+### Sick Picks
+
+- CJ: [Ergonomics](https://en.wikipedia.org/wiki/Ergonomics), [More on Ergonomics](https://www.cuh.nhs.uk/patient-information/seating-and-ergonomics/)
+- Scott: [Project Farm YouTube trash bag comparison](https://www.youtube.com/watch?v=uw9SMpywYHY&ab_channel=ProjectFarm)
+- Wes: [Kitchy Pizza Cutter Wheel](https://www.amazon.ca/dp/B019S3W8AO?crid=2318NXG9JQ4PC&keywords=kitchy+pizza+cutter+wheel&sprefix=Kitchy+,aps,235&language=en_US&dib_tag=se&ref_=as_li_ss_tl&dib=eyJ2IjoiMSJ9.5GOxDTt8x_TajE-EzbID_c9K9tecJLxRGEpMbjRKq2uRGa6jCWrIqUVuSLQxXEoJmugLZrrVyal-RKhqaV8Vg7fPKPlwJ07msnHCGWjlHVJDeWr3TG5ELGpFqpc4JkxeiPELWuQaCxLayq9UIwjONNLt4QGYc3lVK4SqmKjpJ-9-LVtIKEq_CYmQOXmczsFXJhQYl69pftezvdwyhpQoUD9hBZ0K3I4oRXsqUzSZpd4XLrdnB1Vp1iee1Ny7i-42JzoW2M-s0pwUX71iROhHvAS7spVhBxKPnPxr2-NxPM0.rjqi8ZaItYx9mGugQmudhgcaoxZ3nP_EwNzyLRWr5LM&sr=8-5&linkCode=gs2&linkId=dcdd68d1c4400e58c0cdb10527761659&tag=isi777-20)
```

Comment:
This link contains a bunch of tracking and affiliate stuff. Unless this is Wes' or Syntax's affiliate link, you can simplify the URL to just be:

https://www.amazon.ca/dp/B019S3W8AO

Reply:
Oh great, thanks CJ